tw-NEDa-backend/src/main/resources/neda.yaml

```diff
@@ -9,8 +9,8 @@ spring:
      name: neda # Service name
   datasource:
     url: jdbc:oracle:thin:@localhost:1521:xe
-    username: STUDENT
-    password: STUDENT
+    username: neda
```

Comment:
Make you sure we all have the DB setup the same on your laptops.
We all should have a neda user with the neda password.